library/core/src/main/java/com/google/android/exoplayer2/mediacodec/MediaCodecUtil.java

```diff
@@ -318,7 +318,14 @@ private static boolean isCodecUsableDecoder(android.media.MediaCodecInfo info, S
     }
 
     // Work around https://github.com/google/ExoPlayer/issues/4519.
-    if ("OMX.SEC.mp3.dec".equals(name) && "SM-T530".equals(Util.MODEL)) {
+    if (("OMX.SEC.mp3.dec".equals(name) || "OMX.brcm.audio.mp3.decoder".equals(name))
```

Comment:
Presumably each device has one of these decoders or the other. Can the code be structured to reflect this?

Reply:
Oh right, I read the change description and you're saying these devices really do have both of these (and the Google one). Is that definitely true for all of the referenced devices? If so then respond here and otherwise ignore the comment. Thanks!

Reply:
Looking at this more, I don't think it is true that all of these devices can have both of these decoders. "OMX.brcm" is probably Broadcom, but I think only two of these devices have Broadcom chipsets (SM-G350 and GT-I9152). I'd like for the code to accurately reflect what decoders are present on each device.

Reply:
Thank you for catching this out. I am 100% sure about (SM-G350, GT-I9152 and also GT-S7580) because I own psysical devices and debugged on all three them. As you pointed out, other devices (SM-T231, GT-I9152, GT-P5220) were just an assumption of similarity. I've corrected the PR to meet your suggestions and looking forward for round two of review.

PS: Correct me if I'm wrong but I think these codecs should be just somehow prioritized (not just Collections.singletonList(decoderInfos.get(0))) - I thought of this solution as a temporary fix that will be replaced somewhere in the future, as the list of if's may/will continue growing forever with current approach.